tests/by-util/test_dd.rs

```diff
@@ -1165,12 +1165,12 @@ fn test_bytes_iseek_bytes_iflag() {
 }
 
 #[test]
-fn test_bytes_iseek_skip_additive() {
+fn test_bytes_iseek_skip_not_additive() {
```

Comment:
This is the change described in your comment here: https://github.com/uutils/coreutils/pull/3256#issuecomment-1246863326 Looks good